### ISSUE TITLE
Add support for CloudSQL MySQL insights

### ISSUE THREAD

**Issue by bizrad** [terraform-google-modules/terraform-google-sql-db#347]
### TL;DR

Google has added query insights to the MySQL offering as well now.
Please add support for this in the mysql and safer_mysql modules.
Similar to #198 which was for postgres.

### Additional information

Example detected changes when enabling this through the UI on an existing database using default options.
```
  # module.my-sql.module.safer_mysql.google_sql_database_instance.default has changed
  ~ resource "google_sql_database_instance" "default" {
        id                            = "my-db-a1a1a1a1"
        name                          = "my-db-a1a1a1a1"
        # (11 unchanged attributes hidden)

      ~ settings {
          ~ version               = 68 -> 71
            # (9 unchanged attributes hidden)

          + insights_config {
              + query_insights_enabled  = true
              + query_string_length     = 1024
              + record_application_tags = false
              + record_client_address   = false
            }

            # (4 unchanged blocks hidden)
        }

        # (1 unchanged block hidden)
    }
```

**Comment by trajan0x** [comment]
+1 on this

**Comment by kpocius** [comment]
This has been fixed and rolled out in `13.0.0` -- the issue can be closed.

**Comment by apeabody** [comment]
Correct, Thanks!
